src/TestStack.BDDfy/Processors/UnusedExampleException.cs

```diff
@@ -1,21 +1,12 @@
 ﻿namespace TestStack.BDDfy.Processors
 {
     using System;
-    using System.Runtime.Serialization;
 
-    [Serializable]
```

Comment:
You could polyfill this if you wanted. Removing for everyone means you can no longer take exception across appdomains